frontend/src/pages/GraphPF/components/groupExpanded.tsx

```diff
@@ -217,7 +217,7 @@ const BaseGroupExpandedComponent: React.FunctionComponent<BaseGroupExpandedProps
   // "knows" it needs to go one level higher to reach the proper grouping.
   const raise = e => {
     let target = e.target;
-    while (d3.select(target).attr('class') !== 'pf-topology__group') {
+    while (!d3.select(target).attr('class')?.startsWith('pf-topology__group ')) {
       target = target.parentNode;
     }
     d3.select(target.parentNode).raise();
```

Comment:
```suggestion
    try {
      while (d3.select(target).attr('class') !== 'pf-topology__group') {
        target = target.parentNode;
      }
      d3.select(target.parentNode).raise();
    } catch (err) {
      // ignore when this logic doesn't work and travels too far up the chain
    }
```

Unfortunately the change stops the mentioned console error, but also breaks the "raise" feature.  I think maybe we can keep the original code but catch and ignore errors when it doesn't find what it wants.

Reply:
Thanks for the suggestion @jshaughn. As you say, this is a hacky piece of code and I don't understand deeply what the 'raise' feature. In my case I just take `node.tsx`code as reference:

(https://github.com/kiali/kiali/blob/d6f9027e56125a3a031d0fd916f160820fa32152/frontend/src/pages/GraphPF/components/node.tsx#L323C1-L333C5)

Reply:
We should probably add a try/catch in the other location as well.  It took me some time to get the node and group impls to work as expected, and you are right, they are slightly different.  But both are hacks.  The `raise` is basically re-ordering the d3 layers so that the user can see a partially occluded label.  This would not be necessary if PFT didn't overlap labels...